src/devices/third_reality.ts

```diff
@@ -209,6 +221,17 @@ const definitions: DefinitionWithExtend[] = [
         extend: [battery({voltage: true}), forcePowerSource({powerSource: 'Battery'})],
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['3RSM0147Z'],
+        model: '3RSM0147Z',
+        vendor: 'Third Reality',
+        description: 'Soil sensor',
+        fromZigbee: [fz.temperature, fz.humidity],
+        toZigbee: [],
+        exposes: [e.temperature(), e.humidity()],
+        extend: [battery({voltage: true}), forcePowerSource({powerSource: 'Battery'})],
```

Comment:
```suggestion
        extend: [battery({voltage: true}), forcePowerSource({powerSource: 'Battery'}), temperature(), humidity()],
```

And remove exposes, fromZigbee and toZigbee

Reply:
It has been modified. Please check it again.，thanks